starter-code/french-onion-soup-starter-code/css/main.css

```diff
@@ -7,3 +7,37 @@ html {
   font-family: 'Open Sans', sans-serif;
   font-size: 18px
 }
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+#apple-header{
+  background-image: url("../images/soup.jpg");
+  background-size: cover;
+  height: 600px;
+}
+h1 {
+  color: white;
+  font-size: 60px;
+  text-align: center;
+  position: relative;
+  top: 50%;
+}
+
+h2 {
+  font-size: 30px;
+}
+#container{
```

Comment:
Si vais a usar un identificador único, suele ser mejor darle un nombre representativo del contenido mismo. Pensar que containers hay muchos, por lo que o lo dejáis como clase para reutilizarla en el futuro o ponéis un id que sea appletext o algo del estilo.